### DISCUSSION TITLE
Regarding aaptOptions

### DISCUSSION THREAD

**Discussion by soruppuAswin** [google/filament#5907] [Q&A]
Hi,
We are using the following aaptOptions in our project. What does it actually mean with respect to ktx? What impact does it have on the app bundle and .apk size?

```
    aaptOptions {
        noCompress 'filamat', 'ktx'
    }
```

**Comment by romainguy** [comment]
It means ktx files won't be compressed in the APK (basically: no zip). You don't have to do it this way, but leaving the files uncompressed allows for faster loading/decoding. It's really up to you.

**Reply by soruppuAswin** [reply]
Got it! Thank you very much!

**Reply by romainguy** [reply]
More specifically, using uncompressed assets allows you do avoid copies during loading/decoding (using the `Asset` API). Otherwise you first need to read the data into intermediate buffers.
